sql/_13_issues/_24_2h/cases/cbrd_25323.sql

```diff
@@ -0,0 +1,158 @@
+-- This test case verifies CBRD-25323 issue.
+-- The 'Eliminate INNER JOIN' optimization is performed for the first table, and the join type for the second table is removed. 
+-- If the join type for the second table is OUTER JOIN, the 'Eliminate INNER JOIN' optimization is not performed for the first table.
+
+drop table if exists tb_child;
+drop table if exists ta_parent;
+drop table if exists tc_dummy;
+
+select '' as "test data";
+create table ta_parent (p_ca int primary key, p_cb int);
+create table tb_child (c_ca int primary key, c_cb int);
+alter table tb_child add c_parent_ca int not null references ta_parent (p_ca);
+create table tc_dummy (d_ca int primary key, d_cb int);
+
+insert into ta_parent values (1, 1);
+insert into tb_child values (2, 2, 1);
+insert into tb_child values (3, 3, 1);
+insert into tc_dummy values (2, 20);
+insert into tc_dummy values (4, 40);
+
+update statistics on ta_parent with fullscan;
+update statistics on tb_child with fullscan;
+update statistics on tc_dummy with fullscan;
+
+
+select 'AS-IS Eliminate INNER JOIN optimization is not performed.
+TO-BE
+- The join type of the second table is LEFT OUTER JOIN
+- With ANSI JOIN syntax' as "test case 7";
```

Comment:
select 대신에 evaluate를 사용하는 것은 어떤가요?
select는 불필요한 queryPlan을 출력하게 만듭니다.

```suggestion
evaluate 'AS-IS Eliminate INNER JOIN optimization is not performed.
TO-BE
- The join type of the second table is LEFT OUTER JOIN
- With ANSI JOIN syntax' as "test case 7";
```

Reply:
리뷰 반영되었습니다.